plugins/node/opentelemetry-instrumentation-graphql/src/utils.ts

```diff
@@ -296,9 +296,11 @@ export function wrapFields(
   tracer: api.Tracer,
   getConfig: () => GraphQLInstrumentationParsedConfig
 ): void {
+  const config = getConfig();
   if (
     !type ||
     typeof type.getFields !== 'function' ||
+    config.ignoreFields.includes(type.name) ||
```

Comment:
when you instantiate class you could create a hash map of all keys to ingore using `includes` everywhere which is slow.